src/app/home/info-pilots/info-pilots.component.html

```diff
@@ -0,0 +1,13 @@
+
+<div id="infoPilots" class="flex flex-col items-center py-32 px-40">
+    <h2 class="mt-10 mb-40 text-5xl md:text-6xl font-bold text-white text-center">¿Qué Usan los Pilotos?</h2>
+    <div class="lg:grid lg:grid-cols-3 lg:grid-cols-3 gap-5 lg:gap-10">
+        <div *ngFor="let cosa of cosas; index as i"
```

Comment:
Este ngFor no es explicito
Quedaria mas fácil así 
*ngFor="let infoPilot of pilotsData; index as i"